### PR TITLE
Add Google Analytics to homepage

### PR DESCRIPTION
**Pull request by humphd**
Related to https://github.com/mozilla/thimble.webmaker.org/issues/749 and https://github.com/MozillaFoundation/plan/issues/542#issuecomment-135829652

We still need the env updated for the right GA account.

### CODE REVIEW

File: views/homepage/index.html [hunk 1]
```diff
@@ -9,6 +9,14 @@
     <link rel="stylesheet" href="/resources/stylesheets/userbar.css">
     <link rel="stylesheet" href="/homepage/stylesheets/style.css">
     <title>Thimble by Mozilla - An online code editor for learners &amp; educators.</title>
+
+    {% if GA_ACCOUNT %}
```
Comment (sedge) [inline views/homepage/index.html]: You'll need to add this to the options being rendered in 

https://github.com/mozilla/thimble.webmaker.org/blob/bramble/routes/main/homepage.js#L11-L21

Comment (humphd) [inline views/homepage/index.html]: These are defined globally here https://github.com/mozilla/thimble.webmaker.org/blob/bramble/app.js#L125-L126, so should already exist, no?

Comment (sedge) [inline views/homepage/index.html]: Ah, good call.

**Comment by sedge** [comment]
r+

**Pull request merged**